app/templates/createquizsuccess.html

```diff
@@ -0,0 +1,56 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../static/content/createstyle.css">
+{% endblock %}
+{% block content %}
+<div class="create-class-wrapper">
+    <div class="create-container">
+
+        <form action="" method="post" novalidate class="create-form">
+            {{ form.hidden_tag() }}
+            <h1>Create A New Quiz</h1>
+            {{ form.topic(placeholder="Select Topic", class_="center") }}
+            {{ form.qn(size=32, placeholder="Input Question") }}
+            {{ form.op1(size=32, placeholder="Option 1") }}
+            {{ form.op2(size=32, placeholder="Option 2") }}
+            {{ form.op3(size=32, placeholder="Option 3") }}
+            {{ form.op4(size=32, placeholder="Option 4") }}
+            {{ form.corrOp(placeholder="Correct Option") }}
+            {{ form.img }}
+            <a href="{{ url_for('createquiz') }}" class="route2-btn">Add Another Question</a>
+            <a href="{{ url_for('createquizsuccess') }}" class="route2-btn">Submit Quiz</a>
+            <!--<select required>
+                <option value="" disabled selected hidden>Select Topic</option>
+                <option value="">Geometry</option>
+                <option value="">Estimation</option>
+                <option value="">Model</option>
+            </select>
+            <input required type="text" placeholder="Input Question">
+            <input required type="text" placeholder="Option 1">
+            <input required type="text" placeholder="Option 2">
+            <input required type="text" placeholder="Option 3">
+            <input required type="text" placeholder="Option 4">
+            <select required>
+                <option value="" disabled selected hidden>Select Correct Option</option>
+                <option value="">Option 1</option>
+                <option value="">Option 2</option>
+                <option value="">Option 3</option>
+                <option value="">Option 4</option>
+            </select>
+            <button type="button" class="create-btn">Attach Image</button>
+            <button type="submit" class="create-btn">Create Question</button>-->
+        </form>
+    </div>
+</div>
```

Comment:
If this is all the same as in `createquiz.html` would it be better to either extend the template or have an `if-else`?

Reply:
good point